app/build.gradle.kts

```diff
@@ -220,7 +220,6 @@ android {
   }
 
   lintOptions {
-    lintConfig = file("${project.rootDir}/quality/lint.xml")
```

Comment:
Moving this to the root makes it be picked up by default? So we don't need to explicitly set it anymore?

Reply:
Yep